assets/js/modules/idea-hub/components/dashboard/DashboardIdeasWidget/index.js

```diff
@@ -50,34 +55,60 @@ import Empty from './Empty';
 import Footer from './Footer';
 const { useSelect } = Data;
 
-const getHash = ( hash ) => ( hash ? hash.replace( '#', '' ) : false );
-const isValidHash = ( hash ) =>
-	getHash( hash ) in DashboardIdeasWidget.tabToIndex;
-const getIdeaHubContainerOffset = ( ideaHubWidgetOffsetTop ) => {
-	const siteHeaderHeight =
+function getHash( hash ) {
+	return hash ? hash.replace( '#', '' ) : false;
+}
+
+function isValidHash( hash ) {
+	return getHash( hash ) in DashboardIdeasWidget.tabToIndex;
+}
+
+function getIdeaHubContainerOffset( widgetOffset ) {
+	const header =
 		document.querySelector( '.googlesitekit-header' )?.offsetHeight || 0;
-	const adminBarHeight =
-		document.getElementById( 'wpadminbar' )?.offsetHeight || 0;
+	const adminBar = document.getElementById( 'wpadminbar' )?.offsetHeight || 0;
 	const marginBottom = 24;
-	const headerOffset =
-		( siteHeaderHeight + adminBarHeight + marginBottom ) * -1;
-	return ideaHubWidgetOffsetTop + global.window.pageYOffset + headerOffset;
-};
+	const headerOffset = header + adminBar + marginBottom;
 
-const DashboardIdeasWidget = ( {
-	defaultActiveTabIndex,
-	Widget,
-	WidgetReportError,
-} ) => {
-	const ideaHubContainer = useRef();
-	const newIdeas = useSelect( ( select ) =>
-		select( MODULES_IDEA_HUB ).getNewIdeas()
-	);
-	const savedIdeas = useSelect( ( select ) =>
-		select( MODULES_IDEA_HUB ).getSavedIdeas()
-	);
-	const draftIdeas = useSelect( ( select ) =>
-		select( MODULES_IDEA_HUB ).getDraftPostIdeas()
+	return global.window.pageYOffset + widgetOffset - headerOffset;
+}
+
+function DashboardIdeasWidget( props ) {
+	const { defaultActiveTabIndex, Widget, WidgetReportError } = props;
+
+	const [ trackedWidgetView, setTrackedWidgetView ] = useState( false );
+
+	const { hasNoIdeas, hasManyIdeas, savedIdeas, draftIdeas } = useSelect(
```

Comment:
Let's keep the previous style of selection here as the nested select isn't needed here.

Reply:
Updated.